assets/css/adventure.css

```diff
@@ -1,5 +1,17 @@
right:4rem}.md-pb4{padding-bottom:4rem}.md-pl4,.md-px4{padding-left:4rem}.md-px4{padding-right:4rem}.md-py4{padding-top:4rem;padding-bottom:4rem}}@media (min-width:72.06rem){.lg-p0{padding:0}.lg-pt0{padding-top:0}.lg-pr0{padding-right:0}.lg-pb0{padding-bottom:0}.lg-pl0,.lg-px0{padding-left:0}.lg-px0{padding-right:0}.lg-py0{padding-top:0;padding-bottom:0}.lg-p1{padding:1rem}.lg-pt1{padding-top:1rem}.lg-pr1{padding-right:1rem}.lg-pb1{padding-bottom:1rem}.lg-pl1,.lg-px1{padding-left:1rem}.lg-px1{padding-right:1rem}.lg-py1{padding-top:1rem;padding-bottom:1rem}.lg-p2{padding:1.5rem}.lg-pt2{padding-top:1.5rem}.lg-pr2{padding-right:1.5rem}.lg-pb2{padding-bottom:1.5rem}.lg-pl2,.lg-px2{padding-left:1.5rem}.lg-px2{padding-right:1.5rem}.lg-py2{padding-top:1.5rem;padding-bottom:1.5rem}.lg-p3{padding:3rem}.lg-pt3{padding-top:3rem}.lg-pr3{padding-right:3rem}.lg-pb3{padding-bottom:3rem}.lg-pl3,.lg-px3{padding-left:3rem}.lg-px3{padding-right:3rem}.lg-py3{padding-top:3rem;padding-bottom:3rem}.lg-p4{padding:4rem}.lg-pt4{padding-top:4rem}.lg-pr4{padding-right:4rem}.lg-pb4{padding-bottom:4rem}.lg-pl4,.lg-px4{padding-left:4rem}.lg-px4{padding-right:4rem}.lg-py4{padding-top:4rem;padding-bottom:4rem}}body{line-height:1.4}.travel-flip{-webkit-transition:-webkit-transform .1s;transition:-webkit-transform .1s;transition:transform .1s;transition:transform .1s,-webkit-transform .1s;-webkit-transform:rotate(0);transform:rotate(0)}.travel-flip-flipped{-webkit-transform:rotate(180deg);transform:rotate(180deg)}@-webkit-keyframes a{0%{opacity:0;-webkit-transform:translateY(2rem);transform:translateY(2rem)}to{opacity:1;-webkit-transform:none;transform:none}}@keyframes a{0%{opacity:0;-webkit-transform:translateY(2rem);transform:translateY(2rem)}to{opacity:1;-webkit-transform:none;transform:none}}.travel-object-cover,.travel-object-cover>img{-o-object-fit:cover;object-fit:cover}amp-img.rounded img{border-radius:4px}amp-img.circle img{border-radius:50%}.travel-no-focus:focus{outline:none}.travel-type-active,.travel-type-active:visited{color:#fd68ae}.travel-type-artistic,.travel-type-artistic:visited{color:#32dada}.travel-type-drinks,.travel-type-drinks:visited{color:#4e93f8}.travel-type-fashion,.travel-type-fashion:visited{color:#f1c363}.travel-type-food,.travel-type-food:visited{color:#aa5efd}.travel-type-music,.travel-type-music:visited{color:#ec8936}.travel-type-nature,.travel-type-nature:visited{color:#14b52a}.travel-type-nightlife,.travel-type-nightlife:visited{color:#e83a3a}.travel-type-tours,.travel-type-tours:visited{color:#0ece9e}.travel-type-water,.travel-type-water:visited{color:#3abee8}.travel-overflow-container{display:inline-block;min-width:100vw;padding-left:calc((100vw - 72rem) / 2);padding-right:calc((100vw - 72rem) / 2)}.travel-overflow-container>.travel-overflow-wrap{max-width:75rem;margin:0 auto}.travel-link,.travel-link:visited{color:#8b58e3;text-decoration:none;font-family:inherit;-webkit-transition:color .15s ease-in;transition:color .15s ease-in}.travel-link:
```

Comment:
This diff looks more extensive than it is. I only removed `.comment-form .comment-form-email`
from before the rule:
```css
.comment-form .comment-form-rating{width:100%;padding-right:15px;margin-bottom:-15px;}
```